aws-ecs/secrethub.tf

```diff
@@ -0,0 +1,22 @@
+locals {
+  secrets_dir = "${var.secrets_repo}/${var.env}"
+}
+
+module "secrethub_example_app" {
```

Comment:
Made a local module for the SecretHub AWS service creation that:
- Creates an AWS service account
- Creates a KMS key
- Creates a policy to for allowing to decrypt using the KMS key
- Attaches the policy to the specified role
- Creates access rules on specified SecretHub dirs

If we think this is indeed worth to have a module for this, we should place it in a dedicated GitHub repo somewhere.

Reply:
I think it is worth to make a public module for this. But then we should make the KMS-key an optional variable (so only create one if none is provided).

Reply:
Yes, that makes sense, but if you reuse the KMS key for multiple SecretHub services, it does also make the IAM policy creation that's done in the module redundant. So that would probably need to be conditional as well... 
And in that case, you might as well create those two SecretHub resources yourself.

Reply:
So I'm leaning towards leaving it as is now, so that people that need something that just works right away can use it, and people that want more flexibility can use the module as a reference to build their own stuff.